user-docs/documentation/docs/getting-started/installation.md

```diff
@@ -26,23 +26,22 @@ If you want to take is as an example to start your own project,
 
 - Go to the shared configuration file in `packages/serverless-configuration/src/sharedConfig.ts`
 - Change the `projectName` variable to the name of your project (note that this name must not be too long)
-- Change the dev profile name to one that suits you
-- Search and replace "swarmion-starter" and replace it with the same project name
 - Rename the `swarmion-starter.code-workspace` file at the root of the project to `<projectName>.code-workspace`
+- Change the dev profile name to one that suits you
+```
+export const sharedParams = {
+  dev: { profile: 'swarmion-developer' },
+  ...
+```
 
 ## Install modules
 
 - `nvm use`: use the version of node set in `.nvmrc`;
 - `yarn`: install node dependencies in all packages;
 - `yarn package`: compile the common packages;
+- `yarn test`: run tests in the packages;
 - `yarn upgrade-interactive --latest` in order to bump all dependencies to their latest version. Be careful as it may introduce breaking changes;
```

Comment:
```suggestion
- `yarn upgrade-interactive` in order to bump all dependencies to their latest version. Be careful as it may introduce breaking changes;
```

Reply:
Fixed